packages/compartment-mapper/src/node-modules.js

```diff
@@ -993,8 +1093,10 @@ export const compartmentMapForNodeModules = async (
     dev = false,
     commonDependencies = {},
     policy,
+    policyOverride,
```

Comment:
(should) I disagree with node-modules being aware of the concept of policyOverride. I'm not sure if I like Endo being aware of it at all. Might be just a naming issue, I'm not sure what it is yet. I couldn't find a single example of them being passed in too. IT seems like it's a dedicated pass-through for lavamoat to put things in the context of transform, which can be achieved by `maketransform: ({policyOverride}) => (stuff) => {}`

Reply:
This is a carve-out, yes.  I hesitated to add this and the "create" transform.  These could both live in `@lavamoat/node` instead.